### PR TITLE
Export playlist as JSON

### PR DESCRIPTION
**Pull request by 3r3n-n**
**Summary**
This PR addresses issue [#971](https://github.com/nukeop/nuclear/issues/971), so it implements the functionality for exporting a playlist as a local JSON file.

The button layout was added in the `/packages/app/app/components/PlaylistView/index.tsx` file.
The `exportPlaylist` function was added in the `/packages/app/app/actions/playlists.ts` file.
The `'should export the playlist'` test for this functionality was written in the `/packages/app/app/containers/PlaylistViewContainer/PlaylistViewContainer.test.tsx` file.
The mock object for the test was added to the `/packages/app/app/__mocks__/electron.js` file.
Since the respective button and success and error toasts need their labels, all the files in `/packages/i18n/src/locales/...` were updated so `'export-button'`, `'export-fail-title'`, `'export-success-title'`, `'error-save-file'` and `'playlist-exported'` labels were added (in English) to each file.

I ran `npm test` locally and everything is passing. 

The UI looks like this:

https://user-images.githubusercontent.com/82424931/123503215-2c414900-d617-11eb-87b3-9f0d23b699a0.mov

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -455,6 +455,12 @@
 				"semver": "^6.2.0"
```
Comment (nukeop) [inline package-lock.json:455]: Can you remove the changes to this file?

**Review by nukeop — CHANGES_REQUESTED** [review]

**Comment by nukeop** [comment]
Thanks for your contribution, excellent code. Please remove changes to `package-lock.json` and we will merge it.

File: package-lock.json (hunk 1 above)
Comment (3r3n-n) [reply package-lock.json]: Done :)

Comment (3r3n-n) [reply package-lock.json]: Hi ! 
Just updated this PR with the most recent changes from master.
(I also removed the changes from package-lock.json, now I get no diffs)

**Comment by nukeop** [comment]
Great, thank you. I should have merged your feature before merging the translations.

**Pull request merged**